lib/posix/posix_internal.h

```diff
@@ -31,10 +32,10 @@ enum pthread_state {
 	PTHREAD_DETACHED = PTHREAD_CREATE_DETACHED,
 	/* The thread is running and joinable. */
 	PTHREAD_JOINABLE = PTHREAD_CREATE_JOINABLE,
+	/* A joinable thread exited and its return code is available. */
+	PTHREAD_EXITED,
 	/* The thread structure is unallocated and available for reuse. */
 	PTHREAD_TERMINATED,
```

Comment:
If this constant was set equal to zero, would that allow the structure to not require any initialization before first use?

Reply:
> If this constant was set equal to zero, would that allow the structure to not require any initialization before first use?

I opted to have the CREATE constants align with the states across all libcs